ansible_ai_connect/healthcheck/backends.py

```diff
@@ -103,6 +102,7 @@ def check_status(self):
         if not self.enabled:
             return
 
+        FAUX_COMMERCIAL_USER_ORG_ID = "9999999999"
```

Comment:
It may be nicer to remove `COMMERCIAL` from the test/temp Organisation parameter name?